src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -457,7 +457,7 @@ slots:
     range: EntityReference
     multivalued: true
   match_string:
-    description: Strings that are shared by subj/obj. It is recommended to indicate the 
+    description: String that is shared by subj/obj. It is recommended to indicate the 
```

Comment:
**Changes**
Changed plurality in description for `match_string`.

**Background**
I noticed this while working on a different project and started [a discussion](https://github.com/monarch-initiative/mondo-ingest/issues/355#issuecomment-1688946318) with @matentzn, which led to this PR.